src/agent/Server.h

```diff
@@ -10,6 +13,10 @@ class Server : public QObject
     bool Init();
 private:
     void SendData();
+    void CollectInfoAboutDiscs();
+
+    ProtocolVersion m_protocolVersion = VersionOfProtocol;
```

Comment:
I think we do not need it as member. Server will (I guess) always use one type of protocol, so proper protocol version can be simply put into stream.

Reply:
I can imagine that user in future will be able to choose whether he want to see simple status report or with all data. So this member could do this switch.